report/3.Theory/ObjectRecognition.tex

```diff
@@ -0,0 +1,133 @@
+\section{Object Recognition}
+To be able to figure out where the target is, its location within the video stream has to be determined.
+It is not enough to simply answer the binary question "is a target present?" but rather "where is the target located?", so this information can be processed and in turn move the motors to point the laser at the target.
+As mentioned in section~\ref{sec:obj_tracking}, object recognition algorithms can be tedious to do by hand, and as a result of this, machine learning algorithms will be utilized.
```

Comment:
instead of "....algorithms can be tedious to do by hand....." can we do "....algorithms can be tedious to implement by hand...." eller "write" (atleast something different than do"